runtime/lua/vim/treesitter/query_linter.lua

```diff
@@ -0,0 +1,146 @@
+local namespace = vim.api.nvim_create_namespace('vim.treesitter.query_linter')
+-- those node names exist for every language
+local MAGIC_NODE_NAMES = { '_', 'ERROR' }
```

Comment:
```suggestion
local builtin_nodes = { '_', 'ERROR' }
```
I don't think we use SCREAMING_SNAKE_CASE for constants in Lua (and try to avoid "magic")...

Reply:
YES_WE_DO: https://github.com/neovim/neovim/blob/bad218cd6413dd60a1ed9907f882fbf2e92515f1/runtime/lua/man.lua#L3 . Or at least there is precedent.

Reply:
STILL_DONT_LIKE_IT

(at least the name should be changed)